packages/react-devtools-shared/src/devtools/ContextMenu/useContextMenu.js

```diff
@@ -0,0 +1,32 @@
+import {useContext, useEffect} from 'react';
+import {RegistryContext} from './Contexts';
+
+export default function useContextMenu({data, id, ref}) {
+  const {showMenu} = useContext(RegistryContext);
+
+  useEffect(
+    () => {
+      if (ref.current !== null) {
```

Comment:
Can ref.current change nodes between renders?

Reply:
In our limited use of this hook, it would not be expected to change.